solidity/contracts/ConnextVestingWallet.sol

```diff
@@ -2,35 +2,40 @@
 pragma solidity 0.8.20;
 
 import {VestingWallet, VestingWalletWithCliff} from './VestingWalletWithCliff.sol';
+
+import {Ownable} from '@openzeppelin/contracts/access/Ownable.sol';
+import {Ownable2Step} from '@openzeppelin/contracts/access/Ownable2Step.sol';
 import {IERC20} from '@openzeppelin/contracts/token/ERC20/IERC20.sol';
 
-contract ConnextVestingWallet is VestingWalletWithCliff {
-  address constant BENEFICIARY_ADDRESS = 0x74fEa3FB0eD030e9228026E7F413D66186d3D107;
-  address constant CONNEXT_TOKEN_ADDRESS = 0xFE67A4450907459c3e1FFf623aA927dD4e28c67a;
-
-  uint64 constant ONE_YEAR = 365 days;
-  uint64 constant ONE_MONTH = ONE_YEAR / 12;
-  uint64 constant SEPT_05_2023 = 1_693_872_000;
-
-  /**
-   * @dev Vesting schedule:
-   *      - 1/13 of the tokens will be released after 1 year, starting from Sept 5th 2023
-   *      - 1/13 of the tokens will be released every month after that, for 12 months
-   *
-   *     The equivalent vesting schedule has a 13 months duration, with a 1 month cliff, offsetted to
-   *     start from `Sept 5th 2024 - 1 month`: At Sept 5th 2024 the cliff is triggered unlocking
-   *     1/13 of the tokens, and then 1/13 of the tokens will be linearly unlocked every month after that.
-   */
-  uint64 constant VESTING_OFFSET = ONE_YEAR - ONE_MONTH;
-  uint64 constant VESTING_START_DATE = SEPT_05_2023 + VESTING_OFFSET;
-  uint64 constant VESTING_DURATION = ONE_YEAR + ONE_MONTH;
-  uint64 constant VESTING_CLIFF_DURATION = ONE_MONTH;
-
-  uint256 constant TOTAL_AMOUNT = 24_960_000 ether;
-
-  constructor()
-    VestingWalletWithCliff(BENEFICIARY_ADDRESS, VESTING_START_DATE, VESTING_DURATION, VESTING_CLIFF_DURATION)
-  {}
+/**
+ * @dev Vesting schedule:
+ *      - 1/13 of the tokens will be released after 1 year, starting from _vestingStartDate
+ *      - 1/13 of the tokens will be released every month after that, for 12 months
+ *
+ *     The equivalent vesting schedule has a 13 months duration, with a 1 month cliff, offsetted to
+ *     start from `Sept 5th 2024 - 1 month`: At Sept 5th 2024 the cliff is triggered unlocking
+ *     1/13 of the tokens, and then 1/13 of the tokens will be linearly unlocked every month after that.
```

Comment:
check that this comment specifies `Sept 5 2024`, when actually is dependant on the constructor arguments, so is innacurate, blz use a simpler date then, like `Jan 1st 2023` and explain it from there